e2e/regression_test/settings.no_basal_rates.spec.js

```diff
@@ -0,0 +1,24 @@
+const { setup, match, loopSettings, FilterSettings, SettingDefault, SettingType } = require('../../src/index');
+
+describe('Closed loop is not allowed when settings', () => {
+    beforeAll(async () => {
+        await setup.LaunchLoop();
+    });
+    it('are not applied for basal rates', async () => {
+        let config = {
+            scenario: 'flat_cgm_trace',
+            settings: FilterSettings(SettingDefault, [SettingType.BasalRates])
+        };
+        await loopSettings.Configure(config);
+    });
+    it('should not be in closed loop mode', async () => {
+        await expect(match.loop.Icon()).toHaveLabel('Waiting for first run');
+    });
+    it('should show error that indicates why not in closed loop mode', async () => {
+        await waitFor(match.loop.Icon()).toBeVisible().withTimeout(2000);
```

Comment:
[nit] Per our discussion last week, for the future you may want to consider "hiding" this into a simple function to make it easier for others to code.

Reply:
agreed!